### PR TITLE
*: Unfiy error format

### PR DESCRIPTION
**Pull request by zimulala**
unify errors in some packages.
rename startDDLJob to doDDLJob in ddl package.

### CODE REVIEW

File: infoschema/infoschema.go [hunk 1]
```diff
@@ -31,6 +31,28 @@ import (
 	"github.com/pingcap/tidb/util/types"
 )
 
+var (
+	// ErrDatabaseDropExists returns for dropping a non-existent database.
+	ErrDatabaseDropExists = terror.ClassSchema.New(codeDbDropExists, "database doesn't exist")
```
Comment (shenli) [inline infoschema/infoschema.go]: codeDbDropExists -> codeDatabaseDropExists

**Comment by shenli** [comment]
Rest LGTM

**Comment by shenli** [comment]
LGTM

**Comment by coocood** [comment]
LGTM

**Pull request merged**